### PR TITLE
Enable and fix more warnings

### PR DESCRIPTION
**Pull request by crleona**

### CODE REVIEW

File: LayoutTestBase/Config/LYTConfig.m [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 @implementation LYTConfig
 
-NSUInteger const LYTSaveUnlimitedSnapshotsPerMethod = -1;
+NSUInteger const LYTSaveUnlimitedSnapshotsPerMethod = (NSUInteger)-1;
```
Comment (oliverhu) [inline LayoutTestBase/Config/LYTConfig.m]: NSUInteger -1 ??

File: LayoutTestTests/ConfigTests.m [hunk 2]
```diff
@@ -90,7 +90,7 @@ - (void)testResetDefaultsResetsToExpectedValues {
     XCTAssertEqualObjects(viewClassesRequiringAccessibilityLabelsSets, [LYTConfig sharedInstance].viewClassesRequiringAccessibilityLabels);
     // Testing epsilon values is naturally difficult. This failed on iOS 8 if you do XCTAssertEqual, but this works
     XCTAssertTrue([LYTConfig sharedInstance].cgFloatEpsilon > 0.0000099 && [LYTConfig sharedInstance].cgFloatEpsilon < 0.0000101);
-    XCTAssertEqual(-1, [LYTConfig sharedInstance].snapshotsToSavePerMethod);
+    XCTAssertEqual((NSUInteger)-1, [LYTConfig sharedInstance].snapshotsToSavePerMethod);
```
Comment (oliverhu) [inline LayoutTestTests/ConfigTests.m]: same here

**Review by oliverhu — APPROVED** [review]
thanks for the clean up

File: LayoutTestBase/Config/LYTConfig.m (hunk 1 above)
Comment (crleona) [reply LayoutTestBase/Config/LYTConfig.m]: AKA NSIntegerMax - this just needs to be a distinct value, but there's no reason it can't be an int.

**Pull request merged**